web/src/auth/Provider.js

```diff
@@ -137,7 +137,8 @@ export function getProviderUrl(provider) {
 
     return `${urlObj.protocol}//${host}`;
   } else {
-    return Setting.OtherProviderInfo[provider.category][provider.type].url;
+    const info = Setting.OtherProviderInfo[provider.category][provider.type];
+    if (info) {return info.url;}
```

Comment:
@qianxi0410 why the linter doesn't report this?

Reply:
@kofj you already added new item to `OtherProviderInfo`, why add `if` here?

Reply:
In other cases maybe cause page crash, so we need check it first.

Reply:
Code format should be:

```js
if (info) {
  return info.url;
}
```

And make sure all paths return value

Reply:
resolved